draft-ietf-quic-transport.md

```diff
@@ -5759,11 +5778,15 @@ During the creation of a connection, QUIC only provides protection against
 attack from off the network path.  All QUIC packets contain proof that the
 recipient saw a preceding packet from its peer.
 
-The first mechanism used is the source and destination connection IDs, which are
-required to match those set by a peer.  Except for an Initial and stateless
-reset packets, an endpoint only accepts packets that include a destination
-connection that matches a connection ID the endpoint previously chose.  This is
-the only protection offered for Version Negotiation packets.
+Addresses cannot change during the handshake, so endpoints can discard packets
+that are received on a different network path.
+
+The source and destination connection IDs are the primary means of protection
```

Comment:
s/source and destination connection IDs/Source and Destination Connection IDs/

It is my understanding is that we use CamelCase for these terms.

Reply:
We might have gotten sloppy on the distinction, but I believe in most places we capitalize the field names, but lower-case the concepts.

Reply:
Yup.  I don't think that we have the concept of a "destination connection ID" separate from the field though, so I am now referring to the fields directly.